_summaries/bjorck2018understanding_2.md

```diff
@@ -13,7 +13,49 @@ TODO: Summarize the paper:
 * How well does the paper perform?
 * What interesting variants are explored?
 
+## Goal:
+Clarify vague intuitions about batch normalization through experimentation.
+
+## Background
+Normalizing input data is good and batch normalization extends this concept to intermediate layers. Doing this on batches allows for segnificant speed ups, however no one understands fully why this works.
+
+## Batch Norm and Learning Rate
+Activation and gradients in deep networks without batch normalization are heavy tailed which causes exploding gradients. The solution to this is using small LRs, however using small learning rates causes the issues of:
```

Comment:
I am not sure about this summary, and a little confused about why heavy tailed distribution of initial gradient magnitude will cause exploding gradient. I haven't seen "exploding gradient" in the paper, do you mean exploding means and variances of channels? I guess it will be good to explain more here, and show the reasoning behind it.

Reply:
By exploding gradient I did mean exploding means and variances of channels. I assumed they were the same concept but am not confident enough to keep it, either way exploding means and variances of channels is more in line with the paper :) Additionally added more context.